### ISSUE TITLE
Update The CONTRIBUTING.md File To Provide More Guidance While Contributing.

### ISSUE THREAD

**Issue by 0xull** [ocaml/v2.ocaml.org#1330]
The `CONTRIBUTING.md` file needs more content explaining:
!. The Issue and PR templating when making a PR.
2. How to make changes to the build folder, `ocaml.org`, while working on any issue or bug to visually view the changes on the browser and updating the respective folders outside the ocaml.org folder, (it will also be glad to mention that the `ocaml.org` folder is also listed among the .gitignore files/folder).
3. The skills required to contribute to the `ocaml.org` project and the experience level, if necessary. (this provides clarity and transparency to new contributors joining the community).

I would love to discuss further on this with @patricoferris or any authorised person, and I will be glad to closely work on it with your supervision.

** Aside but I want to suggest that the `README.md` file references the `CONTRIBUTING.md` file which contains more contents regarding the project.

**Comment by avsm** [comment]
All these changes sound good. Please do go ahead and propose a PR with the clarifications you would like to see, and we'll work with you to get those merged.

**Comment by 0xull** [comment]
Thank you so much @avsm.

**Comment by 0xull** [comment]
Hello @avsm, I have completed this, but I'm pending to make a PR because the last pending PR I made some hours ago is yet to be merged and it was made to the `master` branch as you suggested at [#1326](https://github.com/ocaml/ocaml.org/pull/1326) and the current PR is expected to be made to the `staging` branch. These difference in branches would cause errors/conflicts in the repository. So I will wait until my last PR is merged then I will make a new PR to the `staging` branch.
